Automattic/wp-calypso
# Stats Widget: Adjust widget mobile layout

Related to #75107 partially

## Proposed Changes

* Use breakpoint `799px` in line with the dashboard to determine the mobile layout for the design of the Stats Widget.
* Adjust layout in line with the latest design except for the `7 Day Highlights` section.

|Before|After|
|-|-|
|<img width="816" alt="截圖 2023-04-27 上午10 52 57" src="https://user-images.githubusercontent.com/6869813/234747698-10becb4e-fdee-4942-942a-2f493818c62d.png">|<img width="841" alt="截圖 2023-04-27 上午11 29 04" src="https://user-images.githubusercontent.com/6869813/234752415-3e6c6561-d968-4d05-9883-0bebf3ecec18.png">|




## Testing Instructions



* Open your local Jetpack site.
* Check out the branch of the Jetpack to the latest `trunk`.
* Build Jetpack if necessary: `jetpack build plugins/jetpack`.
* Run `STATS_PACKAGE_PATH=/path/to/jetpack/projects/packages/stats-admin yarn dev` under `calypso/apps/odyssey-stats`.
* Navigate to `/wp-admin/index.php`.
* Ensure the mobile layout of the Stats Widget looks reasonable.

## Pre-merge Checklist
